### DISCUSSION TITLE
Few Questions

### DISCUSSION THREAD

**Discussion by cloudcompute** [zitadel/zitadel#5780] [Q&A] [answered]
Hi

a. Using Zitadel in Kubernetes

It has both Helm Charts and Terraform module ([link](https://registry.terraform.io/providers/zitadel/zitadel/latest))

Terraform module is still in alpha.. it has a note stating "Work In Progress". I am looking forward to use Zitadel in Kubernetes, Grateful if you can guide me, how can I?

b. How can I integrate Zitadel with [Nginx Ingress Controller](https://github.com/kubernetes/ingress-nginx) and/or [Traefik Proxy](https://github.com/traefik/traefik) .. is there any working example?

c. Which companies are using it in production?

Thank you

**Comment by hifabienne** [comment, marked as answer]
Hei. 
Terraform can already be used even if it is in Alpha. We want to implement tests for all resources and then if will get the released state. https://github.com/zitadel/terraform-provider-zitadel/issues/85

We di have different documentations for NGINX and Traefik here: https://zitadel.com/docs/self-hosting/manage/reverseproxy/reverse_proxy

You can find some of the companies that are using ZITADEL in production on our customer stories blog posts: https://zitadel.com/blog/tags/successstory

**Comment by cloudcompute** [comment]
@hifabienne 

Thank you for the response.

I already knew about Zitadel. But after reading [this post](https://www.reddit.com/r/selfhosted/comments/ugygmu/help_us_build_the_best_open_source_identity/), I got impressed by Zitadel.

**Reply by hifabienne** [reply]
Great to hear that. 😃
